internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -320,9 +320,29 @@ private ConfigKeys() {
   public static final String ALLOW_CLUSTER_WIPE = "allow.cluster.wipe";
 
   /**
-   * Whether the controller is in Azure fabric. Default is false.
+   * Whether the controller is in a cloud environment. Default is false.
    */
-  public static final String CONTROLLER_IN_AZURE_FABRIC = "controller.in.azure.fabric";
+  public static final String CONTROLLER_CLOUD_ENABLED = "controller.cloud.enabled";
```

Comment:
I think we should keep two different flags to act as the feature flags for controller cluster and storage cluster. We can roll both out separately in that case